.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -0,0 +1,88 @@
+name: Smoke Test with IPv6
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  KIND_VERSION: v0.8.1
+  KIND_CONFIG: .github/kind-config-ipv6.yaml
+  CONFORMANCE_TEMPLATE: examples/kubernetes/connectivity-check/connectivity-check-ipv6.yaml
+  TIMEOUT: 2m
+  LOG_TIME: 30m
+
+jobs:
+  conformance-test-ipv6:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Enable IPv6 in docker
+        run: |
+          # Keep existing cgroup-parent in github action
+          sudo sh -c "echo '{ \"cgroup-parent\": \"/actions_job\", \"ipv6\": true, \"fixed-cidr-v6\": \"2001:db8:1::/64\" }' > /etc/docker/daemon.json"
+          sudo cat /etc/docker/daemon.json
+          sudo ip -6 route add 2001:db8:1::/64 dev docker0
+          sudo sysctl net.ipv6.conf.default.forwarding=1
+          sudo sysctl net.ipv6.conf.all.forwarding=1
+          sudo systemctl restart docker
+
+      - name: Build docker images
+        run: |
+          make docker-image-no-clean
+          make docker-operator-generic-image
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0-rc.1
+        with:
+          version: ${{ env.KIND_VERSION }}
+          config: ${{ env.KIND_CONFIG }}
+
+      - name: Load local images into kind cluster
+        run: |
+          kind load docker-image --name chart-testing cilium/cilium:latest
+          kind load docker-image --name chart-testing cilium/operator-generic:latest
+
+      - name: Install cilium chart
+        run: |
+          helm upgrade -i cilium ./install/kubernetes/cilium \
+            --namespace kube-system \
+            --set global.nodeinit.enabled=true \
+            --set global.kubeProxyReplacement=strict \
+            --set global.pullPolicy=Never \
+            --set global.ipv6.enabled=true \
+            --set global.ipv4.enabled=false \
+            --set global.tunnel=disabled \
+            --set global.autoDirectNodeRoutes=true
+
+          kubectl wait -n kube-system --for=condition=Ready --all pod --timeout=5m
+          # To make sure that cilium CRD is available (default timeout is 5m)
```

Comment:
Are we configuring the default timeout to be 5m or 2m?

Reply:
oh thanks, the comment seems outdated as there was recent change in operator CRD. Let me clean it up.